bundles/mod-mda-EPiano.lv2/EPiano.ttl

```diff
@@ -1,214 +1,203 @@
-        lv2:index 14 ;
-        lv2:symbol "event_in" ;
-        lv2:name "Event In"
-    ] .
+@prefix atom: <http://lv2plug.in/ns/ext/atom#> .
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix lv2: <http://lv2plug.in/ns/lv2core#> .
+@prefix mda: <http://moddevices.com/plugins/mda/> .
+@prefix mod: <http://moddevices.com/ns/mod#> .
+@prefix pg: <http://lv2plug.in/ns/ext/port-groups#> .
+@prefix epp: <http://lv2plug.in/ns/ext/port-props#>.
+@prefix units: <http://lv2plug.in/ns/extensions/units#> .
+@prefix pprops: <http://lv2plug.in/ns/ext/port-props#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+mda:mainIn
+    a pg:StereoGroup ,
+        pg:InputGroup ;
+    lv2:name "Input" ;
+    lv2:symbol "in" .
+
+mda:mainOut
+    a pg:StereoGroup ,
+        pg:OutputGroup ;
+    pg:source mda:mainIn ;
+    lv2:name "Output" ;
+    lv2:symbol "out" .
+
+mda:EPiano
+    a lv2:Plugin ,
+        lv2:InstrumentPlugin ;
+    lv2:project mda: ;
+    lv2:symbol "EPiano" ;
+    doap:name "MDA ePiano" ;
+    doap:license <http://usefulinc.com/doap/licenses/gpl> ;
+    lv2:optionalFeature lv2:hardRTCapable ;
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/urid#map> ;
+    lv2:extensionData <http://kxstudio.sf.net/ns/lv2ext/programs#Interface> ;
+    pg:mainInput mda:mainIn ;
+    pg:mainOutput mda:mainOut ;
+    lv2:minorVersion 0;
+    lv2:microVersion 4;
+    mod:brand "MDA" ;
+    mod:label "ePiano" ;
+
+    rdfs:comment """
+The ePiano is based around 12 carefully sampled and mastered Rhodes© Piano (*) samples.
+Rhodes© Pianos are electrical instruments that were designed in the 40’s but only became popular in the 70’s. Instead of using strings to produce sound, it uses tin metal tines which are then amplified through an electrical pickup.
+
+The modulation parameter is used to set the modulation depth and the modulation mode. If the modulation control is set to a negative value the LFO will be modulating panning. If it's set to a positive value it will modulate the volume, resulting in a classic tremolo effect.
+
+Features:
+Modeled by MDA
+(*) “Other product names modeled in this software are trademarks of their respective companies that do not endorse and are not associated or affiliated with MOD.
+Rhodes© Pianos are a trademark or trade name of another manufacturer and were used merely to identify the product whose sound was reviewed in the creation of this product. All other trademarks are the property of their respective holders.”
+    """ ;
+    lv2:port[
+        a lv2:InputPort ,
+            lv2:ControlPort ;
+        lv2:index 0 ;
+        lv2:name "Envelope Decay" ;
+        lv2:symbol "env_decay" ;
+        lv2:default 8.5 ;
+        lv2:minimum 3.0 ;
+        lv2:maximum 23.0;
+        lv2:portProperty epp:logarithmic;
+        units:unit units:seconds

```

Comment:
`units:seconds` is not a valid type, you want to use `units:s` here.
see http://lv2plug.in/ns/extensions/units#s